src/Label/Label.less

```diff
@@ -1,17 +1,20 @@
 @import (reference) "../less/index";
 
 .Label {
-  .text--caps;
   .text--center;
-  .text--line-height-3;
+  .text--line-height-1;
```

Comment:
best practice for mixins now is to put `()` at the end of them so we should update all the mixins in here